### PR TITLE
Corrected premise to premises

### PR DESCRIPTION
**Pull request by greiginsydney**

### CODE REVIEW

**Comment by scanum** [comment]
@nenonix @JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Skype/SfbServer/deploy/deploy-skype-connectivity.md [hunk 1]
```diff
@@ -93,14 +93,14 @@ Skype for Business Server uses the federation access architecture to support con
...
 
-The site that was formerly used to manually provision federation between Skype for Business on-premise deployments and Skype is no longer necessary and will be shut down on 8/15/2019. Federation with Skype now utilizes federated partner discovery, which is the same mechanism required for federation with Skype for Business Online.
+The site that was formerly used to manually provision federation between Skype for Business on-premises deployments and Skype is no longer necessary and will be shut down on 8/15/2019. Federation with Skype now utilizes federated partner discovery, which is the same mechanism required for federation with Skype for Business Online.
 
-Communication between any on-premise Skype for Business deployment and Skype users via the existing Public IM infrastructure now requires the on-premise edge server configuration to be compatible with Skype for Business Online.
+Communication between any on-premises Skype for Business deployment and Skype users via the existing Public IM infrastructure now requires the on-premises edge server configuration to be compatible with Skype for Business Online.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/deploy-skype-connectivity.md]:
```suggestion
Communication between any on-premises Skype for Business deployment and Skype users via the existing Public IM infrastructure now requires the on-premises Edge Server configuration to be compatible with Skype for Business Online.
```

File: Skype/SfbServer/deploy/deploy-skype-connectivity.md [hunk 2]
```diff
@@ -93,14 +93,14 @@ Skype for Business Server uses the federation access architecture to support con
...
 
 > [!NOTE]
 > No action is needed by most customers, including all deployments that federate with Skype for Business Online.
   
-On-premise deployments are required to publish a Federation DNS SRV record for each domain that they host. Guidance is available in [DNS planning](../plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md#dns-planning). Each domain must resolve by DNS SRV query to an edge server FQDN that satisfies a top-level suffix match of the domain. For example, consider the domain "contoso.com":
+On-premises deployments are required to publish a Federation DNS SRV record for each domain that they host. Guidance is available in [DNS planning](../plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md#dns-planning). Each domain must resolve by DNS SRV query to an edge server FQDN that satisfies a top-level suffix match of the domain. For example, consider the domain "contoso.com":
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/deploy-skype-connectivity.md]:
```suggestion
On-premises deployments are required to publish a Federation DNS SRV record for each domain that they host. Guidance is available in [DNS planning](../plan-your-deployment/edge-server-deployments/edge-environmental-requirements.md#dns-planning). Each domain must resolve by DNS SRV query to an Edge Server FQDN that satisfies a top-level suffix match of the domain. For example, consider the domain "contoso.com":
```

File: Skype/SfbServer/deploy/deploy-skype-connectivity.md [hunk 3]
```diff
@@ -128,7 +128,7 @@ No, since you are already federating with Skype for Business Online, this change
 No. If your edge server proxy settings do not enable federation with the Skype for Business Online hosting provider (sipfed.online.lync.com) then this change will not affect that. However, the same DNS and certificate requirements that apply to federating with Skype for Business Online now also apply to federating with Skype users.
  
 **Our company is large and cannot change its edge configuration due to regulatory/compliance/etc reasons … what can we do?**
-Any on-premise organization that cannot change its edge server configuration as specified should reach out to product support at the earliest opportunity.
+Any on-premises organization that cannot change its edge server configuration as specified should reach out to product support at the earliest opportunity.
```
Comment (JohanFreelancer9) [inline Skype/SfbServer/deploy/deploy-skype-connectivity.md]:
```suggestion
Any on-premises organization that cannot change its Edge Server configuration as specified should reach out to product support at the earliest opportunity.
```

**Comment by scanum** [comment]
@LanaChin PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**